python/pyspark/sql/dataframe.py

```diff
@@ -697,6 +697,83 @@ def subtract(self, other):
         """
         return DataFrame(getattr(self._jdf, "except")(other._jdf), self.sql_ctx)
 
+    def dropna(self, how='any', thresh=None, subset=None):
```

Comment:
I'm actually thinking we should just rename it groupby ... groupBy doesn't buy us much. But either way most of the stuff will be taken care of by auto completion.